docs/technical-docs/issues-workflow/psychopy.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Custom Workflows
+parent: General Issues Workflow
+---
+
+# Custom Workflows
+
+## Overview
+
+Custom Issues workflow for specific projects in the lab where the issues are used for more than just
+posting bugs and feature requests.
+
+Reference the [General Issues Workflow](./index.md) document to get a more in-depth description about the
+workflow, and view additional resources.
+
+Sections:
+
+Labels that will signal the kind of issue to make it easier for contributors to find issues that
+they can help with more easily.
+
+Templates:
+
+Template forms for the issues that will provide context to what information is needed to submit an
+issue.
+
+## PsychoPy Knowledge Share
```

Comment:
We will add additional sections for r and python, right? Not saying you need to add those now, but just want to make sure i understand the layout and organization of this doc. That is, this same page would have separate sections for the r and python knowledge shares? If so, great. No need to build those now though. The current plan is to create those as well, but for now, we dont need to make them, as we may also switch to a combined larger knowledge share with labels for diff categories.